### PR TITLE
Fixed min height for text block

### LINKED ISSUE #3653: Text area blocks lose the contextual edit link if they are empty
# Problem
Text area blocks lose the contextual edit link if they are empty

# Steps to reproduce
Try adding a text area with a few line breaks to a layout and then try to edit the block after saving. 

# Proposed solution
- In `admin.scss` add a `min-height: 10px` or greater rule for text area blocks.

### PR DESCRIPTION
**Pull request by dnkemper**
Closes #3653 

# How to test

### CODE REVIEW

File: docroot/themes/custom/uids_base/scss/admin.scss [hunk 1]
```diff
@@ -1605,6 +1605,11 @@ only screen and (min-resolution: 1.5dppx) {
   border-color: transparent;
 }
 
+// Edit layout for textblock not displaying correctly
+.block-inline-blockuiowa-text-area {
+  min-height: 10px;
```
Comment (cory-skeers) [inline docroot/themes/custom/uids_base/scss/admin.scss]: Could we increase the min-height a bit? The configure button is reachable, but can sometimes overlap other parts where it gets less clear what it's attached to.

image

~40px gives enough space to fully contain the button. Not sure if that's so much space that might make the preview a bit misleading for what is actually in the text block, though.

image

Comment (dnkemper) [reply docroot/themes/custom/uids_base/scss/admin.scss]: Woof! Thank you! IMO the 40px works great!

**Review by cory-skeers — APPROVED** [review]
Looks good and works like a charm!

**Comment by dnkemper** [comment]
@cory-skeers marvelous! Thank you so much!

**Pull request merged**